clang/docs/ReleaseNotes.rst

```diff
@@ -394,6 +394,8 @@ Bug Fixes to C++ Support
   expression references to an entity declared outside of the lambda. (#GH64808)
 - Clang's __builtin_bit_cast will now produce a constant value for records with empty bases. See:
   (#GH82383)
+- Fix an issue where a namespace alias could be defined using a qualified name (all name components
+following the first `::` were ignored). (#GH86122)
```

Comment:
The #GH numbers specifically refer to *issues*; I don’t think you’re supposed to include one if there is no GH issue for it.

Reply:
Good to know! Fixed.